src/icon/Icons.tsx

```diff
@@ -812,6 +824,38 @@ export const CloudUpload = () => (
 );
 
 //D
+export const Datasets = () => (
```

Comment:
```suggestion
export const DatabaseOutline = () => (
```
just to name it a bit more semantically

Reply:
done